files/cron/tar_persistent.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/sh
+
```

Comment:
Can you please add something like 

```bash
if [ -z "$FEEDBACK_DIR" ] || [ -z "$STORAGE_DIR" ]; then
    echo "Please define FEEDBACK_DIR and STORAGE_DIR"
    exit 1
elif [ ! -d "$FEEDBACK_DIR" ] || [ ! -d "$STORAGE_DIR" ]; then
    echo "Make sure the following directories exist:"
    echo "$FEEDBACK_DIR"
    echo "$STORAGE_DIR"
    exit 1
fi
```

Just to be sure we don't accidentally run the script manually in a bad way.